### PR TITLE
feat(title): Added the customize title to every page

### PR DESCRIPTION
**Pull request by Shruti3004**
## Description

In the current implementation, we had the same title `FOSSologyUI` on the same page but now we have implemented the customize title on every page.

## Changes

- Added the React helmet dependency which also has MIT license.
- Created the helper component for title.
- Updated all the titles of pages.

## How to test

Visit to all the routes.

### CODE REVIEW

File: src/components/Title/index.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import React from "react";
+import PropTypes from "prop-types";
+
+// For the title of all the pages
+import { Helmet } from "react-helmet";
+
+const Title = ({ title }) => {
+  return (
+    <Helmet>
+      <title>{title}</title>
```
Comment (GMishx) [inline src/components/Title/index.jsx:28]: Adding ` | FOSSology` will be a nice touch to UI. `·` can also be used in place of `|`.

Comment (GMishx) [reply src/components/Title/index.jsx:28]: Just my opinion :slightly_smiling_face:

Comment (Shruti3004) [reply src/components/Title/index.jsx]: Updated @GMishx, I believe using `|` will be a better option.

**Comment by GMishx** [comment]
Changes looks good but occasionally I am greeted with this
```
Failed to compile

src/components/Title/index.jsx
  Line 23:24:  Unable to resolve path to module 'react-helmet'  import/no-unresolved

Search for the keywords to learn more about each error.

This error occurred during the build time and cannot be dismissed.
```

Any idea? I checked the commit, did `yarn install`, removed the build folder but after navigating few pages, I get this error both in terminal and browser.

**Comment by Shruti3004** [comment]
@GMishx rebased the pr, I have one doubt as soon I am logging, it logged me out are you facing the same issue?

**Comment by GMishx** [comment]
> @GMishx rebased the pr, I have one doubt as soon I am logging, it logged me out are you facing the same issue?

@Shruti3004 please rebase, should've been fixed by #120

**Comment by Shruti3004** [comment]
@GMishx rebased the pr

**Comment by GMishx** [comment]
Just missing title for http://localhost:3000/organize/license/create . Other things are working perfectly :+1:

**Comment by Shruti3004** [comment]
@GMishx Added the title for create candidate license page.

**Review by GMishx — APPROVED** [review]
Reviewed and tested.

**Pull request merged**